packages/cryptoplease/pubspec.lock

```diff
@@ -56,14 +56,14 @@ packages:
       name: auto_route
       url: "https://pub.dartlang.org"
     source: hosted
-    version: "3.2.4"
```

Comment:
Check the changes in these files please. Some versions are downgraded, it shouldn't be the case.